tests/v2/actions/kubeapi/ingresses/ingresses.go

```diff
@@ -0,0 +1,39 @@
+package ingresses
+
+import (
+	"github.com/rancher/shepherd/clients/rancher"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// IngressesGroupVersionResource is the required Group Version Resource for accessing ingresses in a cluster,
+// using the dynamic client.
+var IngressesGroupVersionResource = schema.GroupVersionResource{
+	Group:    "networking.k8s.io",
+	Version:  "v1",
+	Resource: "ingresses",
+}
+
+// GetIngressByName is a helper function that returns the ingress by name in a specific cluster, uses ListIngresses to get the ingress.
+func GetIngressByName(client *rancher.Client, clusterID, namespaceName, ingressName string) (*networkingv1.Ingress, error) {
```

Comment:
extension?

Reply:
I say keep it here. What value is it even bringing? This written to use a list as well, by wouldn't we use a Get call instead? I think it should stay an action. What does anybody else think?